apps/web/src/app/loading.tsx

```diff
@@ -0,0 +1,67 @@
+export default function Loading() {
+  return (
+    <div className="flex mt-[50vh] gap-3 w-screen justify-center">
+      <svg
+        width="24"
+        height="24"
+        stroke="#000"
+        viewBox="0 0 24 24"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g>
+          <circle
+            cx="12"
+            cy="12"
+            r="9.5"
+            fill="none"
+            stroke-width="3"
+            stroke-linecap="round"
+          >
+            <animate
+              attributeName="stroke-dasharray"
+              dur="1.5s"
+              calcMode="spline"
+              values="0 150;42 150;42 150;42 150"
+              keyTimes="0;0.475;0.95;1"
+              keySplines="0.42,0,0.58,1;0.42,0,0.58,1;0.42,0,0.58,1"
+              repeatCount="indefinite"
+            />
+            <animate
+              attributeName="stroke-dashoffset"
+              dur="1.5s"
+              calcMode="spline"
+              values="0;-16;-59;-59"
+              keyTimes="0;0.475;0.95;1"
+              keySplines="0.42,0,0.58,1;0.42,0,0.58,1;0.42,0,0.58,1"
+              repeatCount="indefinite"
+            />
+          </circle>
+          <animateTransform
+            attributeName="transform"
+            type="rotate"
+            dur="2s"
+            values="0 12 12;360 12 12"
+            repeatCount="indefinite"
+          />
+        </g>
+      </svg>
+      {/* <svg
```

Comment:
Comment ini masih kepakek? Kalo enggak, dihapus aja ya

Reply:
kak, katanya conflicts are too complex to resolve in github. aku coba buka di github desktop dapet alert "branchname already exists". berarti resolvenya harus pake command line kak?

Reply:
Kak katanya conflicts too complex to resolve on github, aku resolve pake command line ya?